$OpenBSD: patch-src_utils_cpp,v 1.1.1.1 2013/10/16 18:05:25 abieber Exp $

This is covered in the pull request sent upstream:
	https://github.com/akrennmair/newsbeuter/pull/97

If this pull request is merged, this patch will no longer apply.

--- src/utils.cpp.orig	Tue Aug 27 06:20:39 2013
+++ src/utils.cpp	Mon Oct 14 22:06:09 2013
@@ -274,7 +274,7 @@ std::string utils::convert_text(const std::string& tex
  * of all the Unix-like systems around there, only Linux/glibc seems to 
  * come with a SuSv3-conforming iconv implementation.
  */
-#if !(__linux) && !defined(__GLIBC__) && !defined(__APPLE__)
+#if !(__linux) && !defined(__GLIBC__) && !defined(__APPLE__) && !defined(__OpenBSD__)
 	const char * inbufp;
 #else
 	char * inbufp;
